src/assets/scss/components/_starInput.scss

```diff
@@ -0,0 +1,63 @@
+.star-input {
+    &--active {
+        svg {
+            fill: $accent;
+        }
+        span {
+            color: $white;
+        }
+    }
+    &--highlighted {
+        transform: scale(1.25);
+    }
+    &__label {
+        position: relative;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+        &:focus {
+            svg {
+                fill: $accent;
+                transition: ease-in-out .5s;
```

Comment:
Rather than declaring this on `:focus`, it would be better to have this always be there so that the defocus animation is also smooth